src/commcare_cloud/fab/operations/supervisor.py

```diff
@@ -107,6 +107,15 @@ def restart_formplayer():
     _services_restart()
 
 
+@roles(ROLES_FORMPLAYER)
+def restart_formplayer_if_it_is_running_from_old_release_location():
+    from commcare_cloud.fab.operations.formplayer import \
+        formplayer_is_running_from_old_release_location
+
+    if formplayer_is_running_from_old_release_location():
+        _services_restart()
```

Comment:
This appears to be starting/stopping all services. Does "all services" mean "only formplayer" in this context?

Reply:
This maintains the same blemish the old code has which is it treats machines as either "monolith" (in which case you don't need to restart formplayer separately from "all"), or _just_ formplayer (in which case it restarts "all" as a way to restart just formplayer).

In any case, this PR makes this _not_ happen as long as you've run `deploy formplayer` since the formplayer location change from the last couple days.